src/framework/components/rigid-body/component.js

```diff
@@ -709,22 +698,18 @@ Object.assign(pc, function () {
          * @name pc.RigidBodyComponent#_updateKinematic
          * @description Kinematic objects maintain their own linear and angular velocities. This method updates their transform
```

Comment:
Might be worth updating the description.

Reply:
Done. :)